src/docker/complete/with-tls.yml

```diff
@@ -0,0 +1,62 @@
+services:
+  superlink:
+    command:
+      - --ssl-ca-certfile=certificates/ca.crt
+      - --ssl-certfile=certificates/server.pem
+      - --ssl-keyfile=certificates/server.key
+    secrets:
+      - source: superlink-ca-certfile
+        target: /app/certificates/ca.crt
+      - source: superlink-certfile
+        target: /app/certificates/server.pem
+      - source: superlink-keyfile
+        target: /app/certificates/server.key
+
+  superexec:
+    command:
+      - --executor
+      - flwr.superexec.deployment:executor
+      - --executor-config
+      - superlink="superlink:9091",root-certificates="certificates/superlink-ca.crt"
+      - --ssl-ca-certfile=certificates/ca.crt
+      - --ssl-certfile=certificates/server.pem
+      - --ssl-keyfile=certificates/server.key
+    secrets:
+      - source: superlink-ca-certfile
+        target: /app/certificates/superlink-ca.crt
+      - source: superexec-ca-certfile
+        target: /app/certificates/ca.crt
+      - source: superexec-certfile
+        target: /app/certificates/server.pem
+      - source: superexec-keyfile
+        target: /app/certificates/server.key
+
+  supernode-1: &supernode
+    command:
+      - --superlink
+      - superlink:9092
+      - --root-certificates
+      - certificates/ca.crt
+    secrets:
+      - source: superlink-ca-certfile
+        target: /app/certificates/ca.crt
+
+  supernode-2: *supernode
+
+  # uncomment to enable TLS on another supernode
+  #
+  # supernode-3: *supernode
```

Comment:
@Robert-Steiner Why is there are difference in `supernode-1` and `supernode-2`?

In `compose.yml` the pattern is that we have 1 and 2 and can uncomment 3. Can we have the same pattern here? Maybe I am misreading the way its written and that's already that way here.

Reply:
Docker compose supports [YAML anchors](https://docs.docker.com/reference/compose-file/fragments/#example-2). With `&supernode`, we create an anchor, and the following content can be referenced by writing `*supernode`. it reduces code duplication. In `compose.yml` we can use it because we each node has a different `partition-id`.  

But if it is hard to understand, I can also use the same pattern as in the `compose.yml` file.

Reply:
@Robert-Steiner Now knowing it seems simple but reusing the same pattern might make it easier to understand for non-experienced readers so I'd suggest to do so.